### PR TITLE
Bmakau

### PR DESCRIPTION
**Pull request by bmakau**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Test A
- [x] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by bmakau** [comment]
some end points done working on relationships

**Comment by bmakau** [comment]
Conflicts solved.

File: src/main/java/com/lambdaschool/wellness/model/User.java [hunk 1]
```diff
@@ -43,11 +49,16 @@ public String getLname() {
     // this.lname = lname;
     // }
 
-    public String getFullname() {
-        return this.fname + " " + this.lname;
+    public String getFullname()
+    {
+        return this.fullname = lname + " " + fname;
```
Comment (ckirkwood94) [inline src/main/java/com/lambdaschool/wellness/model/User.java]: Should fullname be first name then last name?

Comment (bmakau) [reply src/main/java/com/lambdaschool/wellness/model/User.java]: yeah i can change it.

Comment (ckirkwood94) [reply src/main/java/com/lambdaschool/wellness/model/User.java]: Once this is changed it looks good to me

**Pull request merged**